### DISCUSSION TITLE
Possible documentation issue for pointer events

### DISCUSSION THREAD

**Discussion by LaurentInSeattle** [AvaloniaUI/Avalonia#15772] [Q&A]
Hi all, 

This doc' page https://docs.avaloniaui.net/docs/concepts/input/pointer talks about PointerEnter and PointerLeave

But the code of InputElement says : 

    // Summary:
    //     Defines the Avalonia.Input.InputElement.**PointerEntered** event.
    public static readonly RoutedEvent<PointerEventArgs> PointerEnteredEvent;

    // Summary:
    //     Defines the Avalonia.Input.InputElement.**PointerExited** event.
    public static readonly RoutedEvent<PointerEventArgs> PointerExitedEvent;

Doc error or different events or ? 

Thanks !

**Comment by stevemonaco** [comment]
Yeah, the docs are wrong here. I'll PR a fix. Please post issues to the docs repo in the future: https://github.com/AvaloniaUI/avalonia-docs

**Reply by LaurentInSeattle** [reply]
Will do.. I am so very new to Avalonia, I was not sure if real issue or not

**Reply by stevemonaco** [reply]
PR has been submitted: https://github.com/AvaloniaUI/avalonia-docs/pull/486

**Reply by LaurentInSeattle** [reply]
The API docs seem to also have the same issue: 
See: https://reference.avaloniaui.net/api/Avalonia.Input/InputElement/ 
Which is weird because these are usually autogenerated from code ?

**Reply by stevemonaco** [reply]
The ref docs shouldn't be used at the moment. There are some technical difficulties in generating them, so they're still stuck on 0.10.x. The source code is the best reference, but current master is quite far ahead of 11.0.10.
